controllers/search_controller.rb

```
@@ -163,11 +163,43 @@ class SearchController < ApplicationController
+        agents = resp.map do |doc|
+          affiliations = Array(doc["affiliations_txt"]).map do |aff_txt|
+            begin
+              parsed = JSON.parse(aff_txt)
+              LinkedData::Models::Agent.read_only(
+                id: parsed["id"],
+                name: parsed["name"],
+                acronym: parsed["acronym"],
+                email: parsed["email"],
+                agentType: parsed["agentType"]
+              )
+            rescue JSON::ParserError => e
```

RuboCop (1.73)

[warning] Useless assignment to variable - `e`.

(Lint/UselessAssignment)